builds/misc/metrics-collector-images-release.yaml

```diff
@@ -4,108 +4,410 @@ pr: none
 variables:
   NugetSecurityAnalysisWarningLevel: warn
   Build.Configuration: release
-  
-jobs:
+
+stages:
+# This stage builds Rocks DB and adds the files to staging directory
```

Comment:
Can you clarify in the comment that this is needed for modules used in the tests, not for the metrics collector module?